lib/vault/vault_setup.sh

```diff
@@ -151,7 +151,7 @@ curl --request POST --header "X-Vault-Token: ${key_root}" --data "{\"value\": \"
 
 ## SPECIAL STORES ACCESS
 # Ingest needs access to the opa store's programs path:
-docker exec $vault sh -c "echo 'path \"opa/programs\" {capabilities = [\"update\", \"read\", \"create\"]}' >> ${ingest}-policy.hcl; echo 'path \"opa/programs/*\" {capabilities = [\"create\", \"update\", \"read\", \"delete\"]}' >> ${ingest}-policy.hcl; echo 'path \"opa/site_roles\" {capabilities = [\"create\", \"update\", \"read\", \"delete\"]}' >> ${ingest}-policy.hcl; echo 'path \"opa/users/*\" {capabilities = [\"create\", \"update\", \"read\", \"delete\"]}' >> ${ingest}-policy.hcl; echo 'path \"opa/pending_users\" {capabilities = [ \"update\", \"read\", \"create\"]}' >> ${ingest}-policy.hcl; echo 'path \"opa/preapproved_users\" {capabilities = [ \"update\", \"read\", \"create\"]}' >> ${ingest}-policy.hcl; vault policy write ${ingest} ${ingest}-policy.hcl"
+docker exec $vault sh -c "echo 'path \"opa/programs\" {capabilities = [\"update\", \"read\", \"create\"]}' >> ${ingest}-policy.hcl; echo 'path \"opa/programs/*\" {capabilities = [\"create\", \"update\", \"read\", \"delete\"]}' >> ${ingest}-policy.hcl; echo 'path \"opa/site_roles\" {capabilities = [\"create\", \"update\", \"read\", \"delete\"]}' >> ${ingest}-policy.hcl; echo 'path \"opa/users/*\" {capabilities = [\"create\", \"update\", \"read\", \"delete\"]}' >> ${ingest}-policy.hcl; echo 'path \"opa/pending_users\" {capabilities = [ \"update\", \"read\", \"create\"]}' >> ${ingest}-policy.hcl; echo 'path \"opa/preapproved_users\" {capabilities = [ \"update\", \"read\", \"create\"]}' >> ${ingest}-policy.hcl; echo 'path \"opa/users\" {capabilities = [\"list\"]}' >> ${ingest}-policy.hcl; vault policy write ${ingest} ${ingest}-policy.hcl"
```

Comment:
Can we split this up somehow? This is a bit hard to read
```suggestion
docker exec $vault sh -c "echo 'path \"opa/programs\" {capabilities = [\"update\", \"read\", \"create\"]}' >> ${ingest}-policy.hcl;"
docker exec $vault sh -c "echo 'path \"opa/programs/*\" {capabilities = [\"create\", \"update\", \"read\", \"delete\"]}' >> ${ingest}-policy.hcl;"
docker exec $vault sh -c "echo 'path \"opa/site_roles\" {capabilities = [\"create\", \"update\", \"read\", \"delete\"]}' >> ${ingest}-policy.hcl;"
docker exec $vault sh -c "echo 'path \"opa/users/*\" {capabilities = [\"create\", \"update\", \"read\", \"delete\"]}' >> ${ingest}-policy.hcl;"
docker exec $vault sh -c "echo 'path \"opa/pending_users\" {capabilities = [ \"update\", \"read\", \"create\"]}' >> ${ingest}-policy.hcl;"
docker exec $vault sh -c "echo 'path \"opa/preapproved_users\" {capabilities = [ \"update\", \"read\", \"create\"]}' >> ${ingest}-policy.hcl;"
docker exec $vault sh -c "echo 'path \"opa/users\" {capabilities = [\"list\"]}' >> ${ingest}-policy.hcl;"
docker exec $vault sh -c "vault policy write ${ingest} ${ingest}-policy.hcl"
```